Documentation/cloud-config-locations.md

```diff
@@ -0,0 +1,19 @@
+# Cloud-Config Locations
+
+Every CoreOS boot Cloud-Config tool looks for a config file to configure your host. Here is a list of locations which are used by the Cloud-Config utility, depending on your CoreOS platform:
+
+* Mount point with [config-2]({{site.baseurl}}/os/docs/latest/config-drive.html#contents-and-format) label. It should contain a `openstack/latest/user_data` relative path. An absolute path inside CoreOS should look like `/media/configvirtfs/openstack/latest/user_data`. Usually used by cloud providers or in VM installations.
```

Comment:
This link is not valid. Don't worry about "{{site.baseurl}}", there is a script which will rewrite the link.